src/program/lwaftr/run_nohw/run_nohw.lua

```diff
@@ -62,9 +62,12 @@ function run(parameters)
    local conf_file, b4_if, inet_if, opts = parse_args(parameters)
    local conf = require('apps.lwaftr.conf').load_lwaftr_config(conf_file)
    local c = config.new()
+   local lwaftr_class =  LwAftr
+   lwaftr_class.config_arg = "conf"
+   local device = next(assert(conf.softwire_config.instance))
 
    -- AFTR
-   config.app(c, "aftr", LwAftr, conf)
+   config.app(c, "aftr", lwaftr_class, {conf=conf, device=device})
```

Comment:
What if instead the lwaftr app just gets a normal snabb-softwire-v2 config, but with only one device?  I.e. computing the lwaftr app's config from the whole machine's config will produce a config with no "instance" section, and the "internal-interface" / "external-interface" sections have been specialized for the device at hand.  WDYT?